docs/gettingstarted/index.rst

```diff
@@ -270,6 +274,7 @@ Getting started with Flocker
    .. tutorial-step-condensed::
 
       Now we are going to use a different deployment configuration to show moving the Redis container with its data volume.
+      The web server will remain deployed on the first host and remain accessible via either host's address.
```

Comment:
Personally, I think the tutorial should end with another clickable link and a screen shot showing that the page has been "visited 2 times"...ie the redis data for the first visit has been moved to the new node.
 * https://github.com/ClusterHQ/flocker/pull/1117#issuecomment-72848241